sdk/core/azure-core/CHANGELOG.md

```diff
@@ -8,29 +8,46 @@
 - Added `Range` type to `Azure::Core::Http` namespace.
 - Added support for long-running operations with `Operation<T>`.
 - Added support for setting a custom transport adapter by implementing the method `std::shared_ptr<HttpTransport> ::AzureSdkGetCustomHttpTransport()`.
+- Added interoperability between `std::chrono::system_clock` types and `DateTime`.
 - Added default constructor to `DateTime`.
 - Added `DateTime` supporting dates since 0001.
 - Added Base64 encoding and decoding utility APIs to the `Azure::Core` namespace available from `azure/core/base64.hpp`.
 - Added `Http::Response<void>` template specialization.
+- Added `GetHeadersAsString()` on the `Request` class.
+- Added a `platform.hpp` header file for defining commonly used OS-specific `#define` constants.
```

Comment:
Is `platform.hpp` meant to be a public header for customers to use, @antkmsft or should it have a `_private` suffix OR be under an `/internal` directory like `log.hpp`?
https://github.com/Azure/azure-sdk-for-cpp/blob/abf4a6bb2954117e7f12f3643cb502cec5362a05/sdk/core/azure-core/inc/azure/core/internal/log.hpp#L8

Reply:
To me, it should be private.
However, I think there might be some public headers including the platform header. And it might not be that bad since the symbols that the platform header is using are general-compiler symbols (like _WIN32)